backend/models/post.model.js

```diff
@@ -52,6 +52,31 @@ const postItem = new mongoose.Schema(
       max: 200,
       required: true,
     },
+    brand: { // brand of item (e.g Ikea, Wayfair...)
+      type: String,
+      max: 100,
+      required: true,
+    },
+    height: { // Height of item
+      type: String,
```

Comment:
Why string and not int/float?

Reply:
I wrote this for now because we do not know what units we are going to use.